app/models/reader-store/reader-store.ts

```diff
@@ -115,6 +122,33 @@ export const ReaderStoreModel = types
     },
   }))
   .actions(self => ({
+    fetchCreatorList: flow(function * () {
+      if (self.isFetchingCreatorList) return
+      self.isFetchingCreatorList = true
+      try {
+        const result: ReaderCreatorsResult = yield self.env.likerLandAPI.fetchReaderCreators()
+        switch (result.kind) {
+          case "ok":
+            self.followingCreators.replace([])
+            result.following.forEach(likerID => {
+              const creator = self.createCreatorFromLikerId(likerID)
+              creator.isFollowing = true
+              self.followingCreators.push(creator)
```

Comment:
Would pushing these creators into a local scope array, then call `self.followingCreators.replace()` better then replacing the prop then push it one by one?

Reply:
https://mobx.js.org/refguide/action.html
> actions will batch mutations and only notify computed values and reactions after the (outer most) action has finished